wikimedia/wikipedia-ios
# [ALT TEXT] Add translations for sprint 1 alt text tasks

**Phabricator:** https://phabricator.wikimedia.org/T370725

### Notes
* This PR adds translations for the target languages
* Chinese and French already got some volunteer translations - I chose to keep them 

### Test Steps
1. Change the device language to a target language
2. Run the app, turn on the alt text dev settings toggle
3. Go through the flow through image recs until you see the prompt
4. Make sure the panel, the article view title, and the first version of the modal sheet are translated
